src/sonic-host-services/scripts/caclmgrd

```diff
@@ -273,6 +274,42 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         return allow_internal_docker_ip_cmds
 
+    def get_loopback_ip_from_loopback_table(self, loopback_table, loopback_name):
+
+        if loopback_table:
+            for key, _ in loopback_table.items():
+                if not _ip_prefix_in_key(key):
+                    continue
+
+                iface_name, iface_cidr = key
+                if iface_name.startswith(loopback_name):
+                    ip_ntwrk = ipaddress.ip_network(iface_cidr, strict=False)
+                    if isinstance(ip_ntwrk, ipaddress.IPv4Network):
+                        ip_addr = ip_ntwrk.network_address
+                        return ip_addr
+
+        return None
+
+    def generate_fwd_traffic_from_host_to_soc(self, namespace):
+
+        fwd_dualtor_grpc_traffic_from_host_to_soc_cmds = []
+        if self.DualToR:
+            loopback_table = self.config_db_map[DEFAULT_NAMESPACE].get_table('LOOPBACK_INTERFACE')
+            loopback_name = 'Loopback3'
+            loopback_address = self.get_loopback_ip_from_loopback_table(loopback_table, loopback_name)
+            if loopback_address is not None:
+                mux_table = self.config_db_map[DEFAULT_NAMESPACE].get_table(self.CONFIG_MUX_CABLE)
+                mux_table_keys = mux_table.keys()
+                for key in mux_table_keys:
+                    kvp = mux_table.get(key)
+                    if 'cable_type' in kvp and kvp['cable_type'] == 'active-active':
+                        fwd_dualtor_grpc_traffic_from_host_to_soc_cmds.append(self.iptables_cmd_ns_prefix[namespace] + 
+                                "iptables -t nat -D POSTROUTING --destination {} -j SNAT --to-source {}".format(kvp['soc_ipv4'], loopback_address))
```

Comment:
I wouldn't recommend deleting and reinstalling the entries as this may be just an overhead with so many rule.  Can we just do one flush of this?

Reply:
Not sure if this is ok ?
```fwd_dualtor_grpc_traffic_from_host_to_soc_cmds.append(self.iptables_cmd_ns_prefix[namespace] + 
                    "iptables -t nat -D POSTROUTING --flush")```
I think the flush option only takes in a chain but does not take the source, destination etc options.